docs/examples/promise/08_read_storage_at/index.js

```diff
@@ -11,7 +11,9 @@ const BOB = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
 const STAKING_ASSET = 16000;
```

Comment:
could query `api.query.genericAsset.stakingAssetId()`

Reply:
I will have that in mind if we go through improving these examples.